src/pages/futurehack-2021.js

```diff
@@ -0,0 +1,462 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
```

Comment:
we've updated this package and it had a name change, should now be `@emotion/react`